marquez_client/file_backend.py

```diff
@@ -39,7 +41,9 @@ def put(self, path, headers, payload):
         put_details['headers'] = headers
         put_details['payload'] = payload
 
-        put_json = json.dumps(put_details)
+        put_json = jsonpickle.encode(
+            value=put_details,
+            fail_safe=Utils.handle_fail)
```

Comment:
It doesn't make sense to me that we use jsonpickle here. 
This changes the format of the log and is totally unecessary.

I understood you wanted to use jsonpickle to serialize the airflow.task_info field.